zulip_bots/zulip_bots/create_bot.py

```diff
@@ -0,0 +1,139 @@
+import argparse
+import os
+from pathlib import Path
+
+DOC_TEMPLATE = """Simple Zulip bot that will respond to any query with a "beep boop".
+
+This is a boilerplate bot that can be used as a template for more
+sophisticated/evolved Zulip bots that can be installed separately.
+"""
+
+
+README_TEMPLATE = """This is a boilerplate package for a Zulip bot that can be installed from pip
+and launched using the `zulip-run-bots` command.
+"""
+
+SETUP_TEMPLATE = """import {bot_name}
+from setuptools import find_packages, setup
+
+package_info = {{
+    "name": "{bot_name}",
+    "version": {bot_name}.__version__,
+    "entry_points": {{
+        "zulip_bots.registry": ["{bot_name}={bot_name}.{bot_name}"],
```

Comment:
Does this working assume the generating directory (default `.`) is actually equal to the bot name that is supplied? If so, it seems cleaner to make the base folder and the nested one relative to the current location?

Reply:
The name of the base directory doesn't matter. We assume that the nested directory and the bot module have the same name as the bot. This is the same practice we have for bots under the `zulip_bots/zulip_bots/bots` directory.

Reply:
Right you are, this was from a quick read rather than test.